docs/charts/index.yaml

```diff
@@ -3,12 +3,12 @@ entries:
   wind-river-cloud-platform-deployment-manager:
   - apiVersion: v1
     appVersion: "1.0"
-    created: "2022-12-15T12:33:43.676988525-05:00"
+    created: "2023-01-13T10:23:42.591490174-05:00"
     description: A Helm chart for the Deployment Manager
-    digest: aa1b91cf45a83f378b52c3885a0828f4f7977c81b637b0182f3bece1359d1d1a
+    digest: 0d2a63de46fe9e6847b968e9f77b8a9615d224c936f4e2bfb48b83451c925fde
     icon: https://www.windriver.com/resources/images/wr-logo-red-2018.png
     name: wind-river-cloud-platform-deployment-manager
     urls:
     - wind-river-cloud-platform-deployment-manager-2.0.10.tgz
     version: 2.0.10
-generated: "2022-12-15T12:33:43.676153918-05:00"
+generated: "2023-01-13T10:23:42.58872577-05:00"
```

Comment:
The gophercloud package is updated in 20230119, but the chart is updated in 20230113, which is not preferred, can you revert your change in the chart and index, and issue a make to re-generate the chart?

Reply:
Regenerated the helm chart. Updated the chart and index